Documentation/EditorConfiguration/QtCreatorConfiguration.md

```diff
@@ -70,7 +70,7 @@ In order to so, create a new file anywhere, for example `license-template.creato
 
 ```
 /*
- * Copyright (c) 2024, the SerenityOS developers.
```

Comment:
I don't think we should modify a copyright statement just by virtue of it having the word "Serenity" in it; the two groups of developers are distinct.

Reply:
This is an example in a documentation file, not an actual one, I believe.

Reply:
You're right! Did not look at the filename closely enough :^)